samples/client/src/main/java/global/covesa/sdk/client/ui/PushUiState.kt

```diff
@@ -6,9 +6,17 @@ import global.covesa.sdk.api.client.push.PushManager
 data class PushUiState(
     val registered: Boolean = false,
     val pushDistributor: String = "",
+    val availableDistributors: List<String> = emptyList(),
+    val selectedDistributor: String = "",
+    val savedDistributor: String = "",
 ) {
     constructor(context: Context) : this(
         registered = PushManager.getAckDistributor(context) != null,
-        pushDistributor =  PushManager.getAckDistributor(context).toString(),
+        pushDistributor = PushManager.getAckDistributor(context).toString(),
+        availableDistributors = PushManager.getDistributors(context),
+        selectedDistributor = if (PushManager.getDistributors(context).size > 1) PushManager.getDistributors(
```

Comment:
If `*Distributor` is moved to `String?`, then selectedDistributor should be initialized to `null` and the UI fallback to `savedDistributor` until the user chose another one